practice/pyramid/site.css

```diff
@@ -0,0 +1,26 @@
+div {
+  border: 1px solid black;
+}
+
+.row {
+  margin: 20px;
+  display: flex;
+  height: 100px;
+  box-sizing: border-box;
+  justify-content: space-between;
+}
+
+div.inner {
```

Comment:
If you're going to apply a class to every element in a certain structural position, just select on the structure instead. `.row > div` gets what you want without needing to string a class all over the HTML.